R/grid_code_paths.R

```diff
@@ -22,10 +22,14 @@ tune_grid_loop <- function(resamples,
     rng
   )
 
-  resamples <- pull_metrics(resamples, results, control)
-  resamples <- pull_notes(resamples, results, control)
-  resamples <- pull_extracts(resamples, results, control)
-  resamples <- pull_predictions(resamples, results, control)
+  # carry out arranging by id before extracting each element of results (#728)
```

Comment:
Previously this sorting happened independently for each of the `pull_*()` functions that call `pulley()`. For those that don't call `pulley()`, the ordering is wrong.